lib/matplotlib/axes/_axes.py

```diff
@@ -2259,7 +2259,7 @@ def barh(self, bottom, width, height=0.8, left=None, **kwargs):
             dictionary of kwargs to be passed to errorbar method. `ecolor` and
             `capsize` may be specified here rather than as independent kwargs.
 
-        align : ['edge' | 'center'], optional, default: 'edge'
+        align : ['edge' | 'center'], optional
```

Comment:
@QuLogic  Done. The direction words are really fallible and I checked several time and being afraid to make mistakes.